### DISCUSSION TITLE
ReferenceError: $RefreshSig$ is not defined in Reactjs jsx component

### DISCUSSION THREAD

**Discussion by coderbahrom** [vitest-dev/vitest#4438] [Q&A]
when I test my component it gest like that failed error 
![Screenshot 2023-11-04 at 15 32 52](https://github.com/vitest-dev/vitest/assets/92739868/59b99685-6525-402d-93f2-3361d1ba2293)

**Comment by radqut** [comment]
Try this -> https://github.com/vitest-dev/vitest/issues/4012#issuecomment-1707522924
